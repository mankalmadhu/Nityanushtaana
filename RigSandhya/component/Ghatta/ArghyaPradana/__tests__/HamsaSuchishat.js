import React from 'react';
import ಹ೦ಸಶ್ಶುಚಿಷತ್ from '../HamsaSuchishat';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');

test("ಹ೦ಸಶ್ಶುಚಿಷತ್", () => {
  expect(Renderer.create(
    <ಹ೦ಸಶ್ಶುಚಿಷತ್ />
  ).toJSON()).toMatchSnapshot();

});