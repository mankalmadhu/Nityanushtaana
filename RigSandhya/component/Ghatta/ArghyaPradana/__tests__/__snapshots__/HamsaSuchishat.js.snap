// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಹ೦ಸಶ್ಶುಚಿಷತ್ 1`] = `
<View>
  <ಪೂರ್ವಾ೦ಗ>
    ಹ೦ಸಃ ಶುಚಿಷತ್ ಇತಿ, ಏಕಸ್ಯ ಋಚೋ, ಗೌತಮ ಪುತ್ರೋ ವಾಮದೇವ ಋಷಿಃ, ಸೂರ್ಯೋ ದೇವತಾ ಜಗತೀಛ೦ದಃ ಅರ್ಘ್ಯ ಪ್ರದಾನೇ ವಿನಿಯೋಗಃ ।
  </ಪೂರ್ವಾ೦ಗ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಓ೦ ಹ॒೦ಸಃ ಶು॑ಚಿ॒ಷದ್ವಸು॑ರ೦ತರಿಕ್ಷ॒ ಸದ್ಧೋತಾ᳚ವೇದಿ॒ಷದತಿ॑ಥಿರ್ದುರೋಣ॒ಸತ್ । ನೃ॒ಷದ್ವ॑ರ॒ಸದೃ॑ತ॒ಸದ್ವ್ಯೋ᳚ಮ॒ಸದ॒ಬ್ಜಾ ಗೋ॒ಜಾ ಋ॑ತ॒ಜಾ ಅ॑ದ್ರಿ॒ಜಾ ಋ॒ತಮ್ ।
  </ವಿಷಯ>
</View>
`;
